### DISCUSSION TITLE
ALIST是否有lite版开发计划？

### DISCUSSION THREAD

**Discussion by lamaged** [AlistGo/alist#6217] [Ideas]
感谢alist开发团队提供了这么好的软件，目前在小米AX6路由中安装alist 3.28 已经使用了一个月，整体来说非常好。
由于路由器安装空间限制无法安装3.3以上版本，开发团队有没有计划开发一个"lite"版，将软件做的尽可能小。
考虑到是路由器安装可以删除下面的功能或模块：
1、文件任务中的离线下载全部删除；
2、文件预览全部删除；
3、其他可以删除的功能
4、如果软件依然比较大，删除所有文件管理等功能，只保留网盘挂载。
如果团队需要测试人员，本人可以全力配合。
目前可以测试的环境：
1、红米AX6路由；
2、UBUNTU 小主机；
3、windows 系统。

**Comment by pongfcnkl** [comment]
路由器没必要跑这个
买个玩客云都比路由器强

**Reply by lamaged** [reply]
路由器确实受限，谢谢！

**Comment by anwen-anyi** [comment]
自己编译吧

**Reply by lamaged** [reply]
谢谢安稳，目前正在学go，看看能不能自己编译出来一个^_^

**Comment by AspirantH** [comment]
本人x86路由器，可玩性比较高，arm的相对尴尬😓

**Reply by lamaged** [reply]
确实是，我自己的J4125的单网口小机器是风扇散热，担心长时间工作稳定性变差，如果每次使用的时候开又比较麻烦，装在路由器上比较省事。但是路由器确实太受限，可用空间只有20M，alist 3.28是用UPX压缩后安装的，安装完成剩余空间很少，所以才有了lite版的想法。谢谢！

**Comment by viqbgrg** [comment]
装到tmp分区
